Core/Src/dma2d.c

```diff
@@ -39,12 +40,20 @@ void MX_DMA2D_Init(void)
   /* USER CODE END DMA2D_Init 1 */
   hdma2d.Instance = DMA2D;
   hdma2d.Init.Mode = DMA2D_M2M;
-  hdma2d.Init.ColorMode = DMA2D_OUTPUT_RGB565;
+  hdma2d.Init.ColorMode = DMA2D_OUTPUT_RGB565; /* LCD is RGB565 */
   hdma2d.Init.OutputOffset = 0;
   hdma2d.Init.BytesSwap = DMA2D_BYTES_REGULAR;
   hdma2d.Init.LineOffsetMode = DMA2D_LOM_PIXELS;
   hdma2d.LayerCfg[1].InputOffset = 0;
+#if LV_COLOR_DEPTH == 32
```

Comment:
Are we editing a generated file? If so, can we change this setting in an LVGL file later?